outputs.tf

```diff
@@ -18,11 +18,20 @@ output "vision_url" {
   description = "IBM Visual Insights URL"
 }
 
+output "vision_username" {
+  value = "admin"
+  description = "IBM Visual Insights username to login"
+}
 output "vision_password" {
   value = "${random_password.vision_password.result}"
   description = "IBM Visual Insights 'admin' password"
 }
 
+output "vpc_id" {
+  value = "${ibm_is_vpc.vpc.id}"
+  description = "VPC ID to be used with frontend application"
```

Comment:
No - this might not be used just with a particular set of technical enablement

reword this to something else stating what this really is, e.g. `ID of the VPC created for this application`